frameworks/shared/caller.py

```diff
@@ -123,7 +123,7 @@ def run_in_venv(caller_file, script_file: str, *args,
             PYTHONPATH=os.pathsep.join([
                 rconfig().root_dir,
             ]),
-            AMLB_PATH=os.path.join(rconfig().root_dir, "amlb"),
+            AMLB_PATH=os.path.join(rconfig().root_dir),
```

Comment:
What changed here?

Reply:
I changed the semantic of this variable for https://github.com/openml/automlbenchmark/pull/453/files#diff-9a256fecc9fc69ecce8bcb888ebab5292a2b17233713e2c883cd97d36b86f166

user to point to `automlbenchmark/amlb`, now points to root (`automlbenchmark`)